apps_product_creator/models/odoo_module_version.py

```diff
@@ -0,0 +1,25 @@
+# Copyright (C) 2017-Today: Odoo Community Association (OCA), BizzAppDev
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import models, api
+
+
+class OdooModuleVersion(models.Model):
+    _inherit = 'odoo.module.version'
+
+    @api.multi
+    def _process_clean_module_version(self):
+        for module_version in self:
+            product = self.env['product.product'].search(
+                [('odoo_module_version_id', '=', module_version.id)])
+            product.write({'active': False})
+            all_variant_archived = all(
```

Comment:
Since you are making 3 iterations(all, for, mapped) here I think you should use something with less performance impact:
```python
active_variants = product_obj.search_count(
    [('product_tmpl_id', '=', product.product_tmpl_id.id),
     ('active', '=', True)])
if not active_variants:
    product.product_tmpl_id.write({'active': False, 'website_published': False})
```
what do you think?

Reply:
@bizzappdev any feedback on this?

Reply:
@oscarolar thank you for your suggestion, You are right, I have changed the code as per your suggestion.
cc @elicoidal

Reply:
I do not see the change here. Did I miss somethign?

Reply:
something strange with github this still shows the outdated changes. 
https://github.com/OCA/apps-store/pull/36/files#diff-87908d4d380713ee148592d4aa5e5069R16
please find the changes it is there already